core/store/migrate/migrations/0064_dynamic_fee_txes.sql

```diff
@@ -0,0 +1,35 @@
+-- +goose Up
+
+ALTER TABLE eth_txes ADD COLUMN access_list jsonb;
+ALTER TABLE eth_tx_attempts
+	ADD COLUMN tx_type smallint NOT NULL DEFAULT 0,
+	ADD COLUMN gas_tip_cap numeric(78,0),
+	ADD COLUMN gas_fee_cap numeric(78,0),
+	ADD CONSTRAINT chk_tx_type_is_byte CHECK (
+		tx_type >= 0 AND tx_type <= 255
+	),
+	ADD CONSTRAINT chk_legacy_or_dynamic CHECK (
```

Comment:
worth it to write a little migration test that goes up to 63, adds a bunch of ethtxes/attempts and then migrates to 64 and adds a mix of legacy and dynamic ethtxes/attempts?

Reply:
We don't have migration tests any more, and I think not having them is the right way to go.

With that having been said, I have run some manual tests on a real database and it works ok.

Reply:
I think it definitely makes sense to not run the migration tests in the CI, but we still want to have them for documentation and reproducibility of manual tests IMO